tools/releaser/breaking_changes/v20240805003.md

```diff
@@ -0,0 +1,23 @@
+# Breaking Changes
+
+## SDK changes
+
+go.mongodb.org/atlas-sdk/v20240805003/admin
+Incompatible changes:
+
+- EncryptionAtRestUsingCustomerKeyManagementApi.CreateEncryptionAtRestPrivateEndpoint: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.CreateEncryptionAtRestPrivateEndpointExecute: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.CreateEncryptionAtRestPrivateEndpointWithParams: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.GetEncryptionAtRestPrivateEndpoint: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.GetEncryptionAtRestPrivateEndpointExecute: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.GetEncryptionAtRestPrivateEndpointWithParams: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.GetEncryptionAtRestPrivateEndpointsForCloudProvider: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.GetEncryptionAtRestPrivateEndpointsForCloudProviderExecute: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.GetEncryptionAtRestPrivateEndpointsForCloudProviderWithParams: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.RequestEncryptionAtRestPrivateEndpointDeletion: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.RequestEncryptionAtRestPrivateEndpointDeletionExecute: added
+- EncryptionAtRestUsingCustomerKeyManagementApi.RequestEncryptionAtRestPrivateEndpointDeletionWithParams: added
```

Comment:
nit: It is showing new CRUD operations as incompatible changes which would not be accurate. I believe a major version is being generated in this case due to change in `/api/atlas/v2/orgs/{orgId}/invoices/{invoiceId}/lineItems:search` but this is not mentioned.

Reply:
We have general issue with breaking changes that is covered in the jira:
https://jira.mongodb.org/browse/CLOUDP-261727